docker-install/00-why_docker.md

```diff
@@ -0,0 +1,51 @@
+# Why Docker?
+### A brief intro to Docker as a data analysis tool
+
+For this workshop, we will be conducting all of our analyses in [Docker](https://www.docker.com/resources/what-container).
+Here we will introduce you to the strategy of using Docker for data analysis.
+
+Adapting from [this data science article](https://towardsdatascience.com/docker-for-data-scientists-5732501f0ba4) by Bobby Lindsey, have you ever heard or said these complaints?
+
+> “Not sure why it’s not working on your computer, it’s working on mine.”
+
+>“It’s a pain to install everything from scratch for Linux, Windows, and MacOS, and trying to build the same environment for each OS.”
+
+>“Can’t install the package that you used, can you help me out?”
+
+Package and software installation problems are frustrating and can easily snowball
+into day(s) of troubleshooting.
+
+For this workshop, we will require 40+ software packages (depending on how you count it)
+that each have a various number of other packages they are dependent on.
+Because of this, we've opted to have you install one potentially difficult
+thing (Docker) rather than 40+ potentially difficult things.
+
+We can use Docker to transfer our software set-up to you.
+
+![docker_gif](screenshots/docker.gif)
+
+In this illustration, the software set-up you will be getting from us and using
+to run your analyses is called a **container**.
+In the Docker set-up instructions, you build the container based on an **image**
```

Comment:
```suggestion
In the Docker set-up instructions, you run the container based on an **image**
```
☝️ because we use the `docker run` command and not `docker build` this could head off any confusion for someone who decides to look at the Docker documentation or something. I expect this will not happen often.

Reply:
Ah. Good point.